src/gui/mainwindow.cpp

```diff
@@ -326,6 +326,13 @@ MainWindow::MainWindow(QWidget *parent)
     m_ui->actionCheckForUpdates->setVisible(false);
 #endif
 
+    // Certain menu items should reside at specific places on macOS.
+    // Qt partially does it on its own, but updates and different languages require tuning.
```

Comment:
oh you mean that it also breaks when it is translated. Basically Qt heuristics work on the english text.